spec/bourbon/functions/assign_inputs_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "spec_helper"
+
+describe "assign-inputs" do
+  before(:all) do
+    ParserSupport.parse_file("functions/assign-inputs")
+    @text_inputs_list = [
```

Comment:
I do think that's a good suggestion and would be a lot simpler, especially for the smaller tests I have PRs for. This one would be the trickiest, but I'll try to see what it looks like. Since that's a very different question and affects the whole setup, I'll go ahead and merge and put that up separately.